### ISSUE TITLE
'BUNDLE_GEMFILE' is not recognized...

### ISSUE THREAD

**Issue by silveltman** [CloudCannon/vonge-jekyll-bookshop-template#11]
Error when running npm run install-jekyll
`'BUNDLE_GEMFILE' is not recognized as an internal or external command, operable program or batch file.`

Just used the repo as a template and followed the readme guide, but it doesn't work

**Comment by bglw** [comment]
Hi Sil,

Yes the package commands were written assuming they were running on a Unix system — I should make that more clear and/or make it cross-compatible.

You could try replacing `BUNDLE_GEMFILE=site/Gemfile` in your package.json with `$Env:BUNDLE_GEMFILE = "site/Gemfile";` which should work in powershell. 

Alternatively, you could skip the package.json and just run `bundle exec jekyll serve` inside the `site` directory.

Let me know how you get on — I'm largely away today but I will look at making this repo cross platform out of the box this week.

**Comment by silveltman** [comment]
Both solutions didn't work for me. 

For running bundle exec jekyll serve inside the site directory, it gave all kind of errors with bundle install. It brought me straight back to the "before 11ty times", so I decided to not to use jekyll ever again xd. Ruby just isn't for me.

Since it was a very small procject, I didn't feel like fixing this so I ended up developing locally with live previewing in Cloudcannon. This worked for me in the end. 

Thanks anyway for the help!
